ingest/defaults/nextclade_field_map.tsv

```diff
@@ -0,0 +1,28 @@
+# TSV file that is a mapping of column names for Nextclade output TSV
+# The first column should be the original column name of the Nextclade TSV
+# The second column should be the new column name to use in the final metadata TSV
+# Nextclade can have pathogen specific output columns so make sure to check which
+# columns would be useful for your downstream phylogenetic analysis.
+seqName	seqName
+clade	clade
```

Comment:
Would it make sense to include the `polybasic_cleavage_site` output column or should the phylogenetic builds continue to rely on `scripts/annotate-ha-cleavage-site.py`?